NuimoSDK/INuimoController.cs

```diff
@@ -28,9 +28,11 @@ public enum NuimoConnectionState
         Connected,
         Disconnecting,
     }
-
+    
+    [Flags]
     public enum NuimoLedMatrixWriteOption
```

Comment:
Can you please rename this enum to plural `NuimoLedMatrixWriteOption`? It emphasizes the fact that multiple flags are allowed and matches better with the argument name called `options`.